cartographer/mapping_2d/sparse_pose_graph.h

```diff
@@ -65,11 +65,11 @@ class SparsePoseGraph : public mapping::SparsePoseGraph {
   SparsePoseGraph(const SparsePoseGraph&) = delete;
   SparsePoseGraph& operator=(const SparsePoseGraph&) = delete;
 
-  // Adds a new node with 'constant_data' and a 'pose' that will later be
-  // optimized. The 'pose' was determined by scan matching against
-  // 'insertion_submaps.front()' and the scan was inserted into the
-  // 'insertion_submaps'. If 'insertion_submaps.front().finished()' is
-  // 'true', this submap was inserted into for the last time.
+  // Adds a new node with 'constant_data'. Its 'constant_data->local_pose' was
+  // determined by scan matching against 'insertion_submaps.front()' and the
+  // scan was inserted into the 'insertion_submaps'. If
+  // 'insertion_submaps.front().finished()' is 'true', this submap was inserted
+  // into for the last time.
```

Comment:
Word missing "into for"?

Reply:
Not really, it's passive voice. "For the last time" data was inserted into the submap. I could change it to
```
If 'insertion_submaps.front().finished()' is 'true', data was inserted into this submap for the last time.
```
if this is easier to parse?

Reply:
Changed it.

Reply:
Thanks, this I can parse better.